package/docker/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM alpine
 COPY openmrs /etc/bahmni_config/openmrs
 COPY openelis /etc/bahmni_config/openelis
 COPY offline /etc/bahmni_config/offline
+COPY configuration /etc/bahmni_config/masterdata/configuration
```

Comment:
is this required? Please check with Umair and Mohan or the config team .. we didnt need it for other metadata/masterdata

Reply:
yes @angshu ... from this directory, we are copying to ${OPENMRS_APPLICATION_DATA_DIRECTORY}/configuration. From, here, iniz module will load the files.
Reference: 
https://github.com/Bahmni/openmrs-distro-bahmni/blob/master/package/docker/openmrs/bahmni_startup.sh#L14